packages/services/src/view-service/status.ts

```diff
@@ -6,10 +6,9 @@ export const status: Impl['status'] = async (_, ctx) => {
   const { indexedDb } = await services.getWalletServices();
   const latestBlockHeight = await services.querier.tendermint.latestBlockHeight();
   const fullSyncHeight = await indexedDb.getFullSyncHeight();
-  if (!fullSyncHeight) throw new Error('Last block synced not in db');
```

Comment:
It is almost universally accepted that fullSyncHeight may be undefined during genesis sync . So we should not throw an error in this case